src/Concurrency.h

```diff
@@ -83,21 +82,26 @@ class queuing_rw_mutex {
         if ((_reader_count > 0) || (_writer_count > 1)) {
             queue_writer(lock);
         }
+        short rc2 = _reader_count;
```

Comment:
What's the point of these two lines? (and the ones below). `rc` and `wc` are never used.